src/Appwrite/Platform/Modules/Databases/Http/Databases/Transactions/Update.php

```
@@ -0,0 +1,852 @@
+
+                    // Add table/collection IDs to the payload for realtime channels
+                    $payload = $doc->getArrayCopy();
+                    $payload['$tableId'] = $collection->getId();
+                    $payload['$collectionId'] = $collection->getId();
+
+                    $queueForEvents
+                        ->setParam('documentId', $doc->getId())
+                        ->setParam('rowId', $doc->getId())
+                        ->setPayload($payload);
+
+                    $project = $queueForEvents->getProject();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$project'. (undefined)

(UnusedLocalVariable)